Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management_description.md

```diff
@@ -1,10 +1,12 @@
- ## Set Up a Remote Web Server
- To use the Palo Alto Networks PAN-OS EDL Management integration, you need to set up a remote web server.
- 1. Set up a remote server with Apache.
- 2. Generate a pair of SSH keys. Send the private key into the user’s home directory, into “.ssh” folder in the Apache server. 
-    Append it to the “authorized_keys” file.
- 3. Save the private SSH key in Demisto Credentials.
- 4. To verify the location of the document root where the files are stored, run the following command.
-   - **CentOS**: `"httpd -S"` 
-   - **Ubuntu**: `apcahe2 -S"`
+This integration is deprecated, please refer to [PAN-OS EDL](https://xsoar.pan.dev/docs/reference/integrations/edl) to learn more about hosting EDLs in Cortex XSOAR. 
+
+## Set Up a Remote Web Server
+To use the Palo Alto Networks PAN-OS EDL Management integration, you need to set up a remote web server.
+1. Set up a remote server with Apache.
+2. Generate a pair of SSH keys. Send the private key into the user’s home directory, into “.ssh” folder in the Apache server. 
+Append it to the “authorized_keys” file.
+3. Save the private SSH key in Demisto Credentials.
+4. To verify the location of the document root where the files are stored, run the following command.
```

Comment:
not sure what 'document root' means in this context..?

Reply:
usually it is `/var/www/html`, pending the web server.

Reply:
the point was that i don't understand why this step is needed 😅

Reply:
because when you update the file on the web server, you need to know its path, and the path the pan-os device looks at.